src/views/home.tsx

```diff
@@ -21,8 +19,6 @@ export const Home: FC = () => {
       <StudyWithUs />
       <UpcomingCourses />
       <Alumni />
-      <MentorsWanted />
-      <Benefits />
       <Requirements />
```

Comment:
I suppose, we should remove this section too

Reply:
and we don't want to remove `<MentorsWanted />`, do we?